inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -593,13 +600,36 @@ private void highlightTextAndDisplayMessage(AjaxRequestTarget aTarget, LearningR
                 selectedLayer.getObject());
     }
 
-    private void actionRemoveHistoryItem(AjaxRequestTarget aTarget,
-            LearningRecord aRecord)
+    private void actionRemoveHistoryItem(AjaxRequestTarget aTarget, LearningRecord aRecord)
+        throws IOException, AnnotationException
     {
         aTarget.add(mainContainer);
         annotationPage.actionRefreshDocument(aTarget);
         learningRecordService.delete(aRecord);
         learningRecords.detach();
+        if (aRecord.getUserAction().equals(LearningRecordUserAction.ACCEPTED)) {
+            JCas aJcas = casStorageService.readCas(aRecord.getSourceDocument(), aRecord.getUser());
+            if (isAnnotatedInCas(aRecord, aJcas)) {
+                confirmationDialog.setTitleModel(
+                    new StringResourceModel("alSidebar.history.delete.confirmation.title", this));
+                confirmationDialog.setContentModel(
+                    new StringResourceModel("alSidebar.history.delete.confirmation.content", this,
+                        null));
+                confirmationDialog.show(aTarget);
+                confirmationDialog
```

Comment:
Can the confirmation dialog be resized? It looks very big compared to the size of the text.